LICENSE

```diff
@@ -1,190 +0,0 @@
-                                 Apache License
```

Comment:
No more license?

Reply:
Whoa that is weird, not sure what happened